src/assets/icons/AvatarIcon.tsx

```diff
@@ -1,7 +1,13 @@
-export const AvatarIcon = () => (
+export const AvatarIcon = ({
+  width,
+  height,
+}: {
+  width: string;
+  height: string;
+}) => (
   <svg
-    width='38'
-    height='37'
+    width={width}
+    height={height}
     viewBox='0 0 38 37'
```

Comment:
you can use variables for viewBox also

Reply:
if i use variables for viewBox then dimensions(width and height) don't work

Reply:
i tried to write viewBox={`0 0 ${width} ${height}`}. it don't work

Reply:
should be `0 0 ${width} ${height}`, I think

Reply:
sorry, i mean viewBox={`0 0 ${width} ${height}`}

Reply:
i can't write ` here!)

Reply:
i wrapped to ` it